tests/integration/core/test_views.py

```diff
@@ -74,10 +75,17 @@ def test_get(self):
                  'version': 'unknown'}]})
 
 
+@patch('openslides.core.config.config', config)
+@patch('openslides.core.views.config', config)
 class ConfigViewSet(TestCase):
     """
     Tests requests to deal with config variables.
     """
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
```

Comment:
Please add a docstring that Django 1.8 is not supported.

Better: Add a check of the Django version.